site/helm-integration.md

```diff
@@ -367,6 +369,46 @@ undesirable because it makes it hard to specify exactly what you want,
 in the one place; or to read exactly what is being specified, in the
 one place. In other words, it's better to be explicit.
 
+## Rollbacks
+
+From time to time a release made by the Helm operator may fail, it is
+possible to automate the rollback of a failed release by setting
+`.spec.rollback.enable` to `true` on the `HelmRelease` resource.
+
+> **Note:** a successful rollback of a Helm chart containing a 
+> `StatefulSet` resource is known to be tricky, and one of the main
+> reasons automated rollbacks are not enabled by default for all
+> `HelmRelease`s. Verify the manual rollback of your Helm chart does
+> not cause any problems before enabling it.
+
+When enabled, the Helm operator will detect a faulty upgrade and
+perform a rollback, it will not attempt a new upgrade unless it
+detects a change in values and/or the chart.
+
+### Configuration
+
+```yaml
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+# metadata: ...
+spec:
+  rollback:
+    # If set, will perform rollbacks for this release.
```

Comment:
Can you add a new comment line to each field and specify what's the default value if the field is omitted from the spec?

Reply:
As the values I filled in are the defaults, I solved it in a different way.